backend/data_validation/__main__.py

```diff
@@ -0,0 +1,10 @@
+import yaml
+from data_val_2022 import DataValidation2022
+
+validation_by_year = {2022: DataValidation2022}
+path_to_config = "config.yaml"
```

Comment:
Just a note: if this config file  is used in other files as a var, then it'll be better to create a constants file and add it in there. Then we use PATH_TO_CONFIG everywhere it's used